tests/worker/test_reworker.py

```diff
@@ -380,15 +380,15 @@ def assert_running_count_plan_produces_ordered_worker_and_data_events(
 def take_n_events(
     stream: EventStream[E, Any],
     num: int,
-) -> "Future[list[E]]":
+) -> Future[list[E]]:
```

Comment:
@DiamondJoseph was this there for a reason? I forget

Reply:
Them being quoted? None that I can recall. If it's working without then great, but I worry given the forward ref issues we had in blueapi recently (but no forward refs here, causality is maintained).

Reply:
I had to change that otherwise the pre-commit wouldn't pass